client/lib/plugins/notices.jsx

```diff
@@ -970,6 +969,11 @@ export default {
 							}
 						);
 				}
+
+			case 'RECEIVE_PLUGINS':
+				return i18n.translate( 'Error fetching plugins on %(site)s.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **26** times:
`translate( 'Error fetching plugins on %(siteName)s.' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).